edx-platform/pearson-vue-theme/lms/templates/course.html

```diff
@@ -11,38 +11,8 @@
     <header class="course-image">
       <div class="cover-image">
         <img src="${course.course_image_url}" alt="${course.display_name_with_default} ${course.display_number_with_default}" />
-        <div class="learn-more" aria-hidden="true">${_("LEARN MORE")}</div>
       </div>
     </header>
-    <div class="course-info" aria-hidden="true">
```

Comment:
Was it necessary to remove it completely? @luismorenolopera

Reply:
yes, they wanted something minimalist